Sources/Sentry/SentryMetricKitIntegration.m

```diff
@@ -509,7 +509,9 @@ - (SentryStacktrace *)convertMXFramesToSentryStacktrace:(NSEnumerator<SentryMXFr
         uint64_t imageAddress = mxFrame.address - mxFrame.offsetIntoBinaryTextSegment;
         debugMeta.imageAddress = sentry_formatHexAddressUInt64(imageAddress);
 
-        debugMetas[debugMeta.debugID] = debugMeta;
+        if (nil != debugMeta.debugID) {
+            debugMetas[SENTRY_UNWRAP_NULLABLE(NSString, debugMeta.debugID)] = debugMeta;
+        }
```

Comment:
`m`: We should be able to simplify this to
```suggestion
        debugMetas[binaryUUID] = debugMeta;
```

As the `debugMeta.debugID` is basically the binaryUUID.
https://github.com/getsentry/sentry-cocoa/blob/b2c7769cc38bd692aee5c6ea101589600b97a1a7/Sources/Sentry/SentryMetricKitIntegration.m#L499-L512